src/components/DashboardLayout/DashboardFooter.jsx

```
@@ -35,21 +38,48 @@ const Footer = styled("footer", {
   }),
 }));
 
-// const Copyright = styled("span")(({ theme }) => ({
-//   fontWeight: 400,
-//   color: "#9CA3AF",
-// }));
+const Copyright = styled("span")(({}) => ({
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)